### PR TITLE
Random changes

### PR DESCRIPTION
**Pull request by wm4**
Minor fixes/enhancements.

### CODE REVIEW

File: video/mp_image.h [hunk 1]
```diff
@@ -121,8 +121,16 @@ typedef struct mp_image {
     struct AVBufferRef *icc_profile;
     // Closed captions packet, if any (only after decoder)
     struct AVBufferRef *a53_cc;
+    // Other side data we don't care about.
+    struct mp_ff_side_data *ff_side_data;
+    int num_ff_side_data;
 } mp_image_t;
 
+struct mp_ff_side_data {
+    int type;
```
Comment (jeeb) [inline video/mp_image.h]: Not a nack, but thinking if this should be enum AVFrameSideDataType for the shits and giggles. Internally enums are ints as far as I know so the type itself is a-OK.

Comment (wm4) [reply video/mp_image.h]: Not doing this because it would require to include ffmpeg headers in this header.

Comment (jeeb) [reply video/mp_image.h]: Alright. Fine by me. I just saw the AVBufferRef underneath and thought I'd comment :) .

Comment (wm4) [reply video/mp_image.h]: The AVBufferRef works because it's an opaque struct. Unfortunately that doesn't work for enums.

**Comment by jeeb** [comment]
Builds and runs on both 64bit Windows and Android. No meaningful additional warnings received from GCC or clang, either.

**Pull request merged**